static/app/components/events/interfaces/exception.tsx

```diff
@@ -51,6 +60,28 @@ export function Exception({
 
   const stackTraceNotFound = !(data.values ?? []).length;
 
+  const copyableStackTrace =
+    data.values && !isNative
```

Comment:
native platforms appears to have a threaded stacktraces a lot of the time. If that's the case, this component is not even used. For the sake of simplicity, I've just excluded native platforms from this feature - implementing it for native will be significantly more complex since we may need to fetch crash reports (at least for Cocoa).